### ISSUE TITLE
Only one Author

### ISSUE THREAD

**Issue by muschellij2** [r-lib/desc#44]
Using `pkgdown::build_site`, it calls `desc` and returns the error:
```
Error in ensure_authors_at_r(self) : No 'Authors@R' field!
You can create one with $add_author
```
if there is an `Author:` field but not a `Authors@R:` field.  I simply made the field (deleting `Author`) and it went well.  

Just wondering if you support simple, single author fields.

**Comment by muschellij2** [comment]
Is there a workaround to convert Author to Authors@R.

**Comment by gaborcsardi** [comment]
Yeah, `desc` should support `Author`. Do you know how exactly `pkgdown` calls `desc`?

**Comment by muschellij2** [comment]
Here's a MWE.   If the package uses `Author:` and not `Authors@R:`, there is an error

```r
library(desc)

x = "Package: example
Type: Package
Title: Example DESC
Version: 0.0.1
Author: John Muschelli
Maintainer: John Muschelli <muschellij2@gmail.com>
Description: Here is a description.
License: GPL-2
"

tfile = tempfile()
dir.create(tfile)
setwd(tfile)
writeLines(x, con = "DESCRIPTION" )
desc <- description$new()
desc$get_author()
desc$get_authors()
```

This was closed with https://github.com/hadley/pkgdown/issues/357, but that was just a fix for that package (@hadley recommended to make an Authors@R for that `DESCRIPTION`).  In, `pkgdown`, `desc` is called with `pkgdown::pkg_authors` (https://github.com/hadley/pkgdown/blob/7c49ac51d038d440c1f920fe31ea8014f8a42e81/R/build-home-authors.R#L23), which in turn calls `desc::get_authors`.

This call calls `idesc_get_authors` (https://github.com/r-lib/desc/blob/c99ccd87f92854da990dea54d2b4707714e688dd/R/authors-at-r.R#L74), and fails at ensure_authors_at_r (https://github.com/r-lib/desc/blob/c99ccd87f92854da990dea54d2b4707714e688dd/R/authors-at-r.R#L49).

**Comment by gaborcsardi** [comment]
OK, thanks! So `desc::get_authors()` should support a lone 'Author' field I guess. It is just hard to parse that field in general....

**Comment by gaborcsardi** [comment]
Yeah, I am not sure what I could do here, really, considering that the `Authors` field can be anything.

**Comment by gaborcsardi** [comment]
Maybe `pkgdown` could just use `desc::desc_get("Author")` and / or `desc::desc_get("Maintainer")` if `Authors@R` is not available?

**Comment by hadley** [comment]
Or just tell people to use `Authors@R`, which I have been doing ;)

**Comment by gaborcsardi** [comment]
Yeah, I think I'll close this. If packages want to handle `Author` themselves, they can already....

**Comment by kenahoo** [comment]
I'd like to re-open this - I submitted a ticket at https://github.com/r-lib/pkgdown/issues/727 requesting support for the `Author` and `Maintainer` field, which are a standard part of `DESCRIPTION` files.

In my case, lack of support means I can't build `pkgdown` docs for packages that I don't own, when the author hasn't used `Authors@R`.  Only 15% of the packages I attempted (91 of 577) had `Authors@R`, so that's a pretty low yield.

The `desc` package does seem like a good place for an abstracting interface that can extract author information using any of the standard supported fields from `DESCRIPTION`.  I do understand that *parsing* that information may be problematic, but (e.g.) when building docs it seems quite acceptable to just leave the information in its raw form when it can't be parsed easily.

**Comment by kenahoo** [comment]
Here's a reproducible example:

```
x <- "Package: example
Type: Package
Title: Example DESC
Version: 0.0.1
Author: John Doe
Maintainer: John Doe <jdoe@example.com>
Description: Here is a description.
License: GPL-2
"

d <- description$new(text=x)
d$get_authors()    # Fails

# Correct the issue & retry
d$add_author(d$get_maintainer())
d$get_authors()    # Succeeds
```
